src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellItemRenderer.cs

```diff
@@ -406,12 +438,41 @@ protected virtual void OnTabReselected(ShellSection shellSection)
 
 		protected virtual void SetupMenu(IMenu menu, int maxBottomItems, ShellItem shellItem)
 		{
+			if ((_updateMenuItemIcon is not null || _updateMenuItemTitle is not null) &&
+				_menuSetup &&
+				_updateMenuItemSource is not null &&
+				_bottomView.IsAlive() &&
+				_bottomView.IsAttachedToWindow == true)
```

Comment:
Unless IsAttachedToWindows does some magic, we do not need the "=="
```suggestion
				_bottomView.IsAttachedToWindow)
```